### ISSUE TITLE
Support multiple locations for all stores

### ISSUE THREAD

**Issue by Mancubuss** [Recol/DLSS-Updater#42]
In my case I tend to store all my games locally, and they are stored in different locations: some are on HDD, some are on SSD, and this applies to pretty much all stores I own games in: Epic, GOG, Steam. Though with Steam the app seems to automatically locate libraries, it's not the case with other platforms.

**Comment by Recol** [comment]
@Mancubuss Yes, this is due to the integration with how the launchers themselves work.
The long version is as follows:

- I'm able to support multi-drive detection for a single launcher for Steam/Ubisoft as they have proper integration for the registry which makes it possible to extract all the installation locations, which is why this works for those.
- Epic/GOG/EA Play and Battle.net all do _not_ have this, and there's little way to integrate this which does not involve mass scanning of entire drives, which i am not comfortable with doing due to the use of HDD's, et al, other types of drives which have varying seek I/O times from one another.
- Simply put, in order to support this type of thing, i would need to add a level of broad scanning to such a degree as to be invasive to the user, which would be slower than just rerunning the program with the secondary file locations themselves.

Hope that makes sense.

**Comment by Mancubuss** [comment]
I see there's an _internal\dlss_updater\config.ini file with blank [LauncherPaths] values. And the app asks about location of some libraries, in the process, like EA and Epic. I guess it could ask for more locations same way maybe?

**Comment by Recol** [comment]
@Mancubuss This config file will be used in the future, it has been removed from the main branch as this forms a structural change involving adding a GUI - https://github.com/Recol/DLSS-Updater/issues/23 which will support the ability to remember paths once it's added. 

It's a valid idea, technically it's feasible to do with this, i'll assess potentially adding it in once the GUI is live.

**Comment by Mancubuss** [comment]
Cool, sounds good. Thank you!

**Comment by Recol** [comment]
@Mancubuss I've implemented the initial phase for this, support for additional (file paths) will come later on.

**Comment by Recol** [comment]
Technically now implemented - the launchers just are filled in place, so closing this issue.
